### PR TITLE
feat(sign-up-success): SignUpSuccessScreen UX & navigation

### PR DESCRIPTION
**Pull request by akhlivnoy**

### CODE REVIEW

File: app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.styles.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+interface ISignUpSuccessScreenStyles {
+  container: ViewStyle;
+}
+
+export const styles = StyleSheet.create<ISignUpSuccessScreenStyles>({
+  container: {
+    flex: 1,
+  },
+});
```
Comment (emberglaz) [inline app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.styles.ts]: add positioning

File: app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}) => {
+  const { t } = useTranslation();
+  const isChild = route.params?.isChild;
+  const childAvatar = useAppSelector(state => state.cache.auth.child?.avatar);
+  const parentAvatar = useAppSelector(state => state.cache.auth.parent?.avatar);
+
+  const getTextType = () =>
+    isChild
+      ? 'screens.onboarding.sign_up_success.child'
+      : 'screens.onboarding.sign_up_success.parent';
```
Comment (emberglaz) [inline app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.tsx]: change to if else

File: app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+export const SignUpSuccessScreen: React.FC<ISignUpSuccessScreenProps> = ({
+  navigation,
+  route,
+}) => {
+  const { t } = useTranslation();
+  const isChild = route.params?.isChild;
+  const childAvatar = useAppSelector(state => state.cache.auth.child?.avatar);
+  const parentAvatar = useAppSelector(state => state.cache.auth.parent?.avatar);
+
+  const getTextType = () =>
```
Comment (emberglaz) [inline app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.tsx]: rename to `getCorrectLocalizationPath`

**Review by emberglaz — CHANGES_REQUESTED** [review]

**Review by emberglaz — APPROVED** [review]

File: app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.styles.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+export const styles = StyleSheet.create<ISignUpSuccessScreenStyles>({
+  container: {
+    flex: 1,
+    marginHorizontal: 24,
+  },
+  mt113: {
+    marginTop: 113,
+  },
```
Comment (Reykjavik151) [inline app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.styles.ts:17]: rename it properly
the style name shouldn't indicate how much margin is

File: app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    state => state.cache.auth[isChild ? 'child' : 'parent']?.avatar,
+  );
+
+  const getCorrectLocalizationPath = () => {
+    if (isChild) {
+      return 'screens.onboarding.sign_up_success.child';
+    } else {
+      return 'screens.onboarding.sign_up_success.parent';
+    }
+  };
```
Comment (Reykjavik151) [inline app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.tsx]: `useMemo` and `[isChild]` in deps to update it correctly.
save the result to the variable and use it across the render

**Review by Reykjavik151 — CHANGES_REQUESTED** [review]

File: app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.styles.ts (hunk 4 above)
Comment (emberglaz) [reply app/screens/Onboarding/SignUp/SignUpSuccess/SignUpSuccess.styles.ts:17]: Not valid
After discussion, we decided to keep

**Review by Reykjavik151 — APPROVED** [review]

**Pull request merged**